tox.ini

```diff
@@ -44,6 +44,9 @@ envlist =
     # The aws_lambda tests deploy to the real AWS and have their own matrix of Python versions.
     py3.7-aws_lambda
 
+    # The gcp deploy to the real AWS and have their own matrix of Python versions.
+    py3.7-gcp
```

Comment:
@untitaker I've pushed new changes regarding GCP test integration, does this look fine?

Reply:
yes